_src/bundle.css

```diff
@@ -151,7 +151,7 @@ nav {
 }
 
 .dropbtn:hover {
-    background-color: rgb(255, 227, 128);
+    background-color: rgb(255, 255, 255);
```

Comment:
@SavannahRicks So, @HadleighJae and I modified the dropdown menu styling for improved accessibility as we were adding her local attractions page. Basically we increased the opacity and tried changing it to white. Take a look for mobile and see if you agree with it. We did it just to increase the contrast for reading.